npy_array/README.md

```diff
@@ -4,14 +4,26 @@ This is not an official Google product.
 
 ## Overview
 
-Exposes `npy_array::SerializeToString` to serialize `nda::array` (from the
-[array](https://github.com/dsharlet/array) library) instances to the numpy
-array format (https://numpy.org/devdocs/reference/generated/numpy.lib.format.html).
+This library provides two functions:
+
+* `npy_array::SerializeToString` serializes (writes) instances of `nda::array` (from the [array](https://github.com/dsharlet/array) library) to the numpy array ["npy format"](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html).
+* `npy_array::DeserializeFromNpyString` deserializes (reads) data in the numpy array format into instances of `nda::array`.
 
 ## Code structure
 
-* /npy_array : C++ library
-* /tests : Tests for the C++ library
+* /npy_array: C++ library.
+* /tests: Tests for the C++ library.
+
+## Build and test
+
+### Bazel
```

Comment:
Do you need to 'cd' into the repo root first?

Reply:
Done.